static/src/stylesheets/layout/new-header/_menu-group.scss

```diff
@@ -113,10 +113,8 @@
         }
 
         @include mq(desktop) {
-            @supports(display: flex) {
-                flex-direction: row;
-                flex-wrap: nowrap;
-            }
+            flex-direction: row;
```

Comment:
Mh - the seem to fail in this case. Not sure yet, why, but I thought adding these `flow-` properties to browsers who don't understand them don't do any harm, so it's ok not to wrap them.